bound/kt/src/main/kotlin/tbdex/sdk/httpclient/Balances.kt

```diff
@@ -1,10 +1,13 @@
 package tbdex.sdk.httpclient
 
 import tbdex.sdk.resources.Balance
+import tbdex.sdk.rust.SystemArchitecture
 import tbdex.sdk.web5.BearerDid
 import tbdex.sdk.rust.getBalances as rustCoreGetBalances
 
 fun getBalances(pfiDidUri: String, bearerDid: BearerDid): List<Balance> {
+    SystemArchitecture.set() // ensure the sys arch is set for first-time loading
```

Comment:
There must be a way to call this when someone does an import on tbdex-kt.. 

like maybe a wrapper around the entire sdk thats like this:
```
class TbdexWrapper {
    companion object {
        init {
           SystemArchitecture.set()
        }
    }
}
```